### ISSUE TITLE
HAR: Exploratory Data Analysis and Machine Learning tuning 

### ISSUE THREAD

**Issue by tailyer** [synopticon/synquesticon#34]
As a model for HAR has been trained, accuracy 94%. There are several issues that we need to look further into:
**EDA** Plot charts and graphs of clean data and analyze if there are any suspicious patterns.
**Data imbalance and model's overfitting**
Does the trained model overfit on the training set? Does it tend to yield the most frequent class as its predictions?
**Model hyperparameter tuning**: number of layers, number of filters, which optimizer (Adam, SGD) is the best, etc.

**Comment by tailyer** [comment]
Data imbalance and model's overfitting: the existing model tends to be overfitted on the training set. In details, it tends to yield "walk" as it output for the class "stairsDown" (a minority class)
Some technical approach:

- Generate more training data from other participants.
- Synthesize data (flipping z values, etc.)
- Training heuristic (adding dropouts at FC, tuning model architecture)

**Comment by tailyer** [comment]
Features engineering using statistical patterns on 90-signal window frames:
* Percentiles: 20, 40, 60, 80
* Mean, min, max, standard deviation.
